src/components/applicantMain/ApplicantMain.tsx

```diff
@@ -25,9 +30,8 @@ import ResumeFolder from '../resumeFolder/ResumeFolder.tsx'
 import Support from '../support/Support.tsx'
 
 const drawerWidth = 240
```

Comment:
if we move constant, the naming should be in this style: DRAWER_WIDTH

Reply:
https://www.freecodecamp.org/news/when-to-capitalize-your-javascript-constants-4fabc0a4a4c4/

Reply:
Changed naming.